src/bun.js/bindings/bindings.cpp

```diff
@@ -3075,9 +3074,6 @@ void JSC__JSValue__getNameProperty(JSC__JSValue JSValue0, JSC__JSGlobalObject* a
     }
 
     JSC::JSValue name = obj->getDirect(vm, vm.propertyNames->toStringTagSymbol);
```

Comment:
we should probably switch this to getIfPropertyExists

Reply:
Done in https://github.com/oven-sh/bun/pull/1674/commits/76f64875695528d6a5c83ebb88465cdca14e02b0